lesson07.blog-api-server/server.js

```diff
@@ -1,22 +1,37 @@
 const http = require('http');
 const fs = require('fs');
 const url = require('url');
+const  _ = require('./myLodash.min');
+
 
 const contentTypes = new Map();
 contentTypes.set('html', 'text/html');
 contentTypes.set('js', 'text/javascript');
 contentTypes.set('css', 'text/css');
 contentTypes.set('json', 'application/json');
 
-
+const HOME_PAGE = "index.html";
+const HTML = "html";
+const SORT = "sortBy";
 http.createServer(function (req, res) {
-    const reqUrl = url.parse(req.url);
-    const ext = reqUrl.pathname.split('.')[1];
-    const fileName = reqUrl.pathname.substr(1);
+    const reqUrl = url.parse(req.url,true);
+    let ext = reqUrl.pathname.split('.')[1];
```

Comment:
what will happen if `pathname` value is `/index.`?
what will happen if `pathname` value is `/`?